src/main/java/mj/chess/Board.java

```diff
@@ -1,82 +1,117 @@
 package mj.chess;
 
-import mj.chess.pieces.Color;
-import mj.chess.pieces.Pawn;
+import mj.chess.pieces.Piece;
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
 import static java.util.stream.Collectors.joining;
-
+import static mj.chess.utils.StringUtil.appendNewLine;
 
 public class Board {
-    private static final int MAX_SIZE = 8;
-    private static final int SECOND_ROW = 1;
-    private static final int SEVENTH_ROW = 6;
-    private final List<Pawn> pawns = new ArrayList<>();
-    private final StringBuilder rowOfBlackPawns = new StringBuilder();
-    private final StringBuilder rowOfWhitePawns = new StringBuilder();
-    private final String[] cellsOfBoard = {
-            "********",
-            "********",
-            "********",
-            "********",
-            "********",
-            "********",
-            "********",
-            "********"
-    };
+    private int numOfPieces = 0;
+    private final int ROWS = 8;
+
+    private final int FIRST_ROW = 0;
+    private final int SECOND_ROW = 1;
+    private final int THIRD_ROW = 2;
+    private final int FORTH_ROW = 3;
+    private final int FIFTH_ROW = 4;
+    private final int SIXTH_ROW = 5;
+    private final int SEVENTH_ROW = 6;
+    private final int EIGHTH_ROW = 7;
+
+    private final List[] rows = new ArrayList[ROWS];
```

Comment:
의도는 이해하겠으나 가독성이 너무 좋지 않습니다. 
`List`를 리스트로 담을 수 있습니다. 

```suggestion
    private final List<List<Piece>> rows = new ArrayList();
```